libs/community/langchain_community/document_compressors/flashrank_rerank.py

```diff
@@ -69,11 +71,10 @@ def compress_documents(
         final_results = []
 
         for r in rerank_response:
-            metadata = r["meta"]
-            metadata["relevance_score"] = r["score"]
-            doc = Document(
-                page_content=r["text"],
-                metadata=metadata,
-            )
-            final_results.append(doc)
+            if r["score"] >= self.score_threshold:
```

Comment:
is there any possible configuration of `client` that would return negative scores?

Reply:
The scores provided by the client come from a softMax activation function and therefore can only be positive.